.github/workflows/build-image-master.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Build and push Docker image tagged master
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read # principle of least privilege
+
+jobs:
+  build:
+    name: Build and push Docker image tagged master
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up Docker Buildx for multi-platform builds
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: ${{ vars.PLATFORMS }}
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          platforms: ${{ vars.PLATFORMS }}
+          push: true
+          sbom: true
+          tags: >-
+            ${{ secrets.DOCKERHUB_ORGANIZATION_NAME }}/${{ vars.IMAGE_NAME }}:master
```

Comment:
Given that we have a `main` branch not a `master` branch, using the `master` tag feels a little weird.

Reply:
Thanks, @neilcook!

> Given that we have a `main` branch not a `master` branch, using the `master` tag feels a little weird.

I have changed the tag from `master` to `main`.

> I guess this is hard to test without actually pushing to the main branch or tagging a release.

Not to worry, the workflows are passing in the fork (we are about to merge), albeit using alternative variables and secrets, hence pushing to a different repository on Docker Hub.